packages/clerk-js/src/core/constants.ts

```diff
@@ -6,6 +6,7 @@ export const DEV_BROWSER_SSO_JWT_HTTP_HEADER = 'Clerk-Cookie';
 
 export const CLERK_MODAL_STATE = '__clerk_modal_state';
 export const CLERK_SYNCED = '__clerk_synced';
+export const CLERK_SYNCING = '__clerk_syncing';
```

Comment:
❓ Is there a way to avoid the extra parameter?

Reply:
Just for clarification 
- `__clerk_synced` is used by a satellite app in order to prevent syncing from firing more that once & to mark that syncing has been attempted.
- `__clerk_syncing` is expected from a primary app in order to perform the redirection back to the satellite. It will also be used to throw the interstitial so the redirection happens without flickering

When we are constructing the URL that contains `__clerk_syncing=true` we always pair it with a `redirect_url`. 
Do you think it will be sufficient enough to only use the `redirect_url` query param ,that points to the satellite app, for this flow to work ?

Reply:
Yes probably `redirect_url` is not sufficient enough on its own. Let's combine both of them into the new `__clerk_satellite_url` as proposed.